org.eclipse.winery.workflowmodeler/src/app/util/http.service.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2017 ZTE Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v10.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     ZTE - initial API and implementation and/or initial documentation
+ */
+import { Injectable } from '@angular/core';
+import { Http, RequestOptionsArgs } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+import './rxjs-operators';
+
+@Injectable()
+export class HttpService {
+    constructor(private http: Http) {}
+
+    public get(uri: string): Observable<any> {
```

Comment:
Yes you are right, we just car about the response data. But i just realized you are using this service in all your other services right? So you'll get all kind of different classes here.
Don't mind, that's OK.